view/base/index.css

```diff
@@ -8,8 +8,8 @@
 
 :root {
   /* Colors */
-  --surface-0: oklch(96% 0.005 300);
-  --surface-1: oklch(100% 0 0);
+  --surface-0: oklch(96% 0.005 300 / 100%);
```

Comment:
Can you revert it? `100%` is default value

Reply:
Revered!